### DISCUSSION TITLE
Found an unhandled GET request with concatenated url

### DISCUSSION THREAD

**Discussion by unclejustin** [trpc/trpc#5456] [General]
When running vitest locally to test React components I get a bunch of these cluttering up the console.

`Found an unhandled GET request to http://localhost:3000/api/trpc/unifiedMembers.getMemberDetail,unifiedMembers.getMemberAttributes,tasks.getTasksByMemberId,tasks.getLastTouchpoint?batch=1&input=...`

Input truncated for brevity.

tRPC work fine when running in dev and in production. I'm sure I'm missing something simple, but damned if I can find it. Anyone able to point me in the right direction 🙇 

Next 13.4.8 using app dir
tRPC 10.40.0

**Comment by KATT** [comment]
Seems like an msw error or something, maybe use a httpLink instead of httpBatchLink if you're intercepting requests

**Reply by unclejustin** [reply]
Thanks KATT!

Sorry, I'm dumb, this is definitely msw my bad.
